clang/test/Driver/cuda-windows-cuda-path.cpp

```diff
@@ -0,0 +1,13 @@
+///
+/// Make sure that CUDA_PATH is picked up correctly when looking for CUDA
+/// instalation.
+///
+
+// REQUIRES: system-windows
+// RUN: env CUDA_PATH=%S\Inputs\CUDA_111\usr\local\cuda %clang -fsycl \
+// RUN: -fsycl-targets=nvptx64-nvidia-cuda -### -v %s 2>&1 | \
+// RUN: FileCheck %s
+
+// CHECK: Found CUDA installation: {{.*}}Inputs\CUDA_111\usr\local\cuda, version 11.1
+
+int main() {}
```

Comment:
```suggestion

```
I guess the main is not necessary in this test.
BTW, you can consider extending clang/test/Driver/cuda-detect.cu instead.

Reply:
Yeap, not needed and yes will move it to the detection test.